src/Control/Effect/Diagnostics.hs

```diff
@@ -158,19 +158,23 @@ instance Show FailureBundle where
   show = show . renderFailureBundle
 
 renderFailureBundle :: FailureBundle -> Doc AnsiStyle
-renderFailureBundle FailureBundle {..} =
-  vsep
-    [ annotate (color Yellow) "----------",
-      annotate (color Yellow) "An error occurred:",
-      "",
-      indent 4 (renderSomeDiagnostic failureCause),
-      "",
-      ">>>",
-      "",
-      indent 2 (annotate (color Yellow) "Relevant warnings include:"),
-      "",
-      indent 4 (renderWarnings failureWarnings)
+renderFailureBundle FailureBundle{..} =
+  vsep $
+    [ annotate (color Yellow) "----------"
+    , annotate (color Yellow) "An error occurred:"
+    , ""
+    , indent 4 (renderSomeDiagnostic failureCause)
+    , ""
     ]
+    ++ if null failureWarnings
```

Comment:
Change is unrelated to adding yarn support, but this excludes the "Relevant warnings include" addendum to error messages if there are no relevant warnings